packages/seacas/libraries/ioss/src/catalyst/Iocatalyst_DatabaseIO.C

```diff
@@ -808,8 +816,25 @@ namespace Iocatalyst {
     }
 
     template <typename GroupingEntityT>
-    bool readFields(const conduit_cpp::Node &&parent, GroupingEntityT *block) const
+    bool readFields(const conduit_cpp::Node &&parent, GroupingEntityT *block, Ioss::DatabaseIO  *dbase) const
```

Comment:
Could also get database from `block->get_database()` so wouldn't necessarily need this argument.